Dockerfile

```diff
@@ -1,36 +1,33 @@
-FROM node:16 AS node-build
+FROM registry.suse.com/bci/nodejs:14 AS node-build
```

Comment:
@dottorblaster Do you know if this can bring as some potential error?
Moving from node16 backwards to node14 (which I guess is the version packaged by SUSE now)

Reply:
@arbulu89 I don't know if this brings potential errors but I wouldn't go backwards this way, node 16 is the latest LTS and I would try to maintain at least v16. @dcermak do you know if SUSE has node16 images?

Reply:
Not yet, we have a node 16 image in the works, but it hasn't been created & published yet.

Reply:
FYI, the nodejs 16 image will probably be published as a beta release with the 15 SP4 beta

Reply:
@dcermak can we use it just referencing it in the dockerfile when the beta comes out? Or do we need additional work? This would be great!

Reply:
I think we would just have to reference it and that would be it.

Reply:
Perfect, thank you!